content/en/docs/community/release_checklist.md

```diff
@@ -4,44 +4,67 @@ description: "Checklist for maintainers when releasing the next version of Helm.
 weight: 2
 ---
 
-**IMPORTANT**: If your experience deviates from this document, please document
-the changes to keep it up-to-date.
+# A Maintainer's Guide to Releasing Helm
 
-## A Maintainer's Guide to Releasing Helm
-
-So you're in charge of a new release for helm? Cool. Here's what to do...
-
-![TODO: Nothing](../../images/nothing.png)
-
-Just kidding! :trollface:
+Time for a new Helm release! As a Helm maintainer cutting a release, you are
+the best person to [update this
+release checklist](https://github.com/helm/helm-www/blob/master/content/en/docs/community/release_checklist.md)
+should your experiences vary from what's documented here.
```

Comment:
Here I edit this intro from having a joke to giving actionable guidance.

Reply:
Why are we removing humour from an engineering document. I put it in there as this is an engineering document. Is there a particular reason for its removal?

Why can't we have both?

Reply:
My edit is intended to make this document sound more welcoming. If someone reads this particular joke as bait-and-switch, they could find it alienating. The release checklist isn't "nothing" - it's a detailed list of complex steps, which could appear intimidating to someone new to the topic who is trying to learn by reading. Tech documents that make light of complexity can end up looking like exclusionary gatekeeping, which is of course quite far from the Helm community's real approach.

Reply:
okay.